src/components/sidebar/components/sidebar-back-to-link/style.module.css

```diff
@@ -1,4 +1,6 @@
 .backToLink {
+  composes: ds-token-typography-body-base from global;
+  font-weight: var(--ds-token-font-weight-medium);
```

Comment:
Tweaked styles here to match Figma.

Reply:
I've updated the styles for this component in my sidebar PR, could we undo the changes here?

Reply:
For sure! Will review #58 now, and happy to wait till that's merged, then rebase this work. With the `href` > `External link icon` functionality in #58, I might need to add back in the `basePath` work I had here previously, but that should fine for demo purposes, I think 👍

Reply:
Have rebased! I think we still need the `composes: body-base` and `font-weight` updates here to match the latest mockups, so have left those in.

Reply:
👍🏻